e2e_tests/docker_image/Dockerfile

```diff
@@ -0,0 +1,82 @@
+FROM ubuntu:18.04
+
+ENV PKG_CONFIG_PATH /usr/local/lib/pkgconfig
+
+RUN apt update
+RUN apt install -y autoconf-archive libcmocka0 libcmocka-dev procps
+RUN apt install -y iproute2 build-essential git pkg-config gcc libtool automake libssl-dev uthash-dev doxygen libjson-c-dev
+RUN apt install -y --fix-missing wget python3 cmake clang
+RUN apt install -y gcc-multilib
+RUN apt install -y gcc-arm-linux-gnueabihf
+RUN apt install -y gcc-aarch64-linux-gnu
+RUN apt install -y libini-config-dev libcurl4-openssl-dev tpm2-tools curl libgcc1
```

Comment:
Did you by any chance check if all of these are actually needed 😬 ?

Reply:
Yes, I started from scratch - they're all needed 😞